python/packages/ai/teams/ai/planner/openai_planner.py

```diff
@@ -133,7 +140,9 @@ def _add_chat_completion_service(self) -> None:
 
     async def _complete_prompt(
         self,
+        turn_state: TurnState[ConversationT, UserT, TempT],
```

Comment:
this will probably break the typings since the ai class that uses it is using a generic type

Reply:
I created `examples/openai_planner.py` to demonstrate the usage and also configured lint for the `examples` folder to ensure there's no typing issues. Please check that file and let me know what might break it.

Reply:
I see your problem with a new `StateT = TypeVar("StateT", bound=TurnState[ConversationState, UserState, TempState])` declared and used. I will investigate and create new PR for this as mentioned in your PR. Let's merge this PR first.